recipes/libvpx/all/conanfile.py

```diff
@@ -0,0 +1,137 @@
+                'mips': 'mips32',
+                'mips64': 'mips64',
+                'sparc': 'sparc'}.get(str(self.settings.arch))
+        build_compiler = str(self.settings.compiler)
+        if build_compiler == 'Visual Studio':
+            compiler = 'vs' + str(self.settings.compiler.version)
+        elif build_compiler in ['gcc', 'clang', 'apple-clang']:
+            compiler = 'gcc'
+        else:
+            raise ConanInvalidConfiguration("Unsupported compiler '{}'.".format(build_compiler))
+
+        build_os = str(self.settings.os)
+        if build_os == 'Windows':
+            os_name = 'win32' if self.settings.arch == 'x86' else 'win64'
+        elif build_os in ['Macos', 'iOS', 'watchOS', 'tvOS']:
+            os_name = 'darwin11'
+        elif build_os == 'Linux':
+            os_name = 'linux'
+        elif build_os == 'Solaris':
+            os_name = 'solaris'
+        elif build_os == 'Android':
+            os_name = 'android'
+        target = "%s-%s-%s" % (arch, os_name, compiler)
+        if tools.cross_building(self) or self.settings.compiler == 'Visual Studio':
+            args.append('--target=%s' % target)
+        if str(self.settings.arch) in ["x86", "x86_64"]:
+            for name in self._arch_options:
+                if not self.options.get_safe(name):
+                    args.append('--disable-%s' % name)
+        with tools.vcvars(self.settings) if self.settings.compiler == 'Visual Studio' else tools.no_op():
+            env_build = AutoToolsBuildEnvironment(self, win_bash=win_bash)
+            env_build.configure(args=args, configure_dir=self._source_subfolder, host=False, build=False, target=False)
+        return env_build
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        with tools.vcvars(self.settings) if self.settings.compiler == 'Visual Studio' else tools.no_op():
+            env_build = self._configure_autotools()
+            env_build.make()
+
+    def package(self):
+        with tools.chdir(self.build_folder):
+            env_build = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+            env_build.install()
+
+        self.copy(pattern="LICENSE", src=self._source_subfolder, dst='licenses')
+        if self.settings.os == 'Windows' and self.settings.compiler == 'Visual Studio':
+            name = 'vpxmt.lib' if 'MT' in str(self.settings.compiler.runtime) else 'vpxmd.lib'
+            if self.settings.arch == 'x86_64':
+                libdir = os.path.join(self.package_folder, 'lib', 'x64')
+            elif self.settings.arch == 'x86':
+                libdir = os.path.join(self.package_folder, 'lib', 'Win32')
+            shutil.move(os.path.join(libdir, name), os.path.join(self.package_folder, 'lib', 'vpx.lib'))
+        try:
+            shutil.rmtree(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
+        except FileNotFoundError:
+            pass
```

Comment:
could this be replaced by `tool.rmdir` ?

Reply:
maybe

Reply:
```suggestion
        tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
```